loading_NYC_taxi_data.ipynb

```diff
@@ -0,0 +1,854 @@
+{
```

Comment:
<p>If you have time, you should read up on Python style guide: https://www.python.org/dev/peps/pep-0008/</p><p> </p><p>We usually add a space around the assignment operator, e.g.</p><pre class="ql-syntax" spellcheck="false">&nbsp; df_yellow2020[elem]['year'] = df_yellow2020[elem].pick_up_date.dt.year
</pre><p> </p><p>A cleaner way to work with strings: https://realpython.com/python-f-strings/</p><p> </p><p>Instead of <code>yellow_2020url+months[elem]+end</code>,  we can have <code>{yellow_2020url}{months[elem]}.csv</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/harrislam1/covid19_NYC_taxi_trips/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Duly noted.